analyzer/consensus/consensus.go

```diff
@@ -738,6 +739,7 @@ func (m *Main) queueAllowanceChanges(batch *storage.QueryBatch, data *storage.St
 				allowanceChange.Beneficiary.String(),
 			)
 		} else {
+			batch.Queue(allowanceOwnerUpsertQuery, allowanceChange.Owner.String())
```

Comment:
Since this was a little sneaky, let's document it?
```suggestion
			// It is possible to create an allowance even for a brand new account with no funds.  
			// Make sure the account exists in indexer DB so that the `allowances->accounts` foreign key is satisfied.
			batch.Queue(allowanceOwnerUpsertQuery, allowanceChange.Owner.String())
```

Reply:
Added a version of this.